NitroxModel/Packets/ClaimPlayerSlotReservation.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class ClaimPlayerSlotReservation : Packet
+    {
+        public string CorrelationId { get; set; }
```

Comment:
Immutable?  Same with below packets.

Reply:
I couldn't remember which serializer was sensitive to restricted mutators. If that isn't the case with packets then I'm happy to do so.

Reply:
This is again one of our code conventions, aimed towards safety and showing intent (there is no serialisation limitation here, shouldn't be with future protobuf either). Since we are not modifying packets after they are created (in other words, all data required for a packet must be sent to the constructor in one go), we prefer to completely omit the (`private`) `set`ter. That way the auto-property can only be assigned from the constructor.